samples/_opengl/ParticlesBasic/src/ParticlesBasicApp.cpp

```diff
@@ -39,7 +39,7 @@ class BasicParticleApp : public App {
 	bool	isOffscreen( const vec2 &v );
 
 public:
-	static const int	NUM_PARTICLES = 10000;
```

Comment:
This is very heavy for the GPU so I thought it would be nicer to have a fluid sample.

Reply:
Perf. issues are actually due to the CPU issues in Modern-GL and the 'immediate mode' gl wrapper. I think it looks better with 1000 particles and still runs really fast on my machine. Although in general, I think it's best to leave unrelated changes like this to separate PRs.

Reply:
Agreed, I removed it in my new PR #2135